lib/src/native/realm_core.dart

```diff
@@ -633,6 +681,130 @@ class _RealmCore {
       return RealmAppCredentialsHandle._(_realmLib.realm_app_credentials_new_email_password(emailPtr, passwordPtr.ref));
     });
   }
+
+  RealmHttpTransportHandle createHttpTransport(HttpClient httpClient) {
+    return RealmHttpTransportHandle._(_realmLib.realm_http_transport_new(
+      Pointer.fromFunction(request_callback),
+      httpClient.toGCHandle(),
+      nullptr,
+    ));
```

Comment:
```suggestion
    final httpPtr = _realmLib.invokeGetPointer(() => _realmLib.realm_http_transport_new(
      Pointer.fromFunction(request_callback),
      httpClient.toGCHandle(),
      nullptr,
    ), "Error when creating http transport");
    return RealmHttpTransportHandle._(httpPtr);
```

Reply:
this can not fail so no invokeGetPointer is needed.